dataeng/jobs/analytics/SnowflakeMetricCollection.groovy

```diff
@@ -0,0 +1,75 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeQueueDepth {
+
+    public static def job = { dslFactory, allVars ->
+
+        Map SnowflakeWarehouseCreditConfig = [
+            NAME: "collect_credit_metrics"
```

Comment:
These job names might be a little vague out of context.  I wouldn't be surprised we have over 100 jobs already---we need the job name to set itself apart a little.  What about:

* snowflake-metrics-collect-cedits
* snowflake-metrics-collect-queue-depth

Just adding "snowflake" might be sufficient.  Also, looking at other job names, it seems we typically use dashes instead of underscores.